iron/distribution.yaml

```diff
@@ -1731,6 +1731,24 @@ repositories:
       url: https://github.com/borglab/gtsam.git
       version: develop
     status: developed
+  gz_ros2_control:
+    doc:
+      type: git
+      url: https://github.com/ros-controls/gz_ros2_control.git
+      version: master
+    release:
+      packages:
+      - gz_ros2_control
+      - gz_ros2_control_demos
+      tags:
+        release: release/iron/{package}/{version}
+      url: https://github.com/ros2-gbp/ign_ros2_control-release.git
```

Comment:
@ahcorde is this the right release repo? I notice it still has "ign" in the name. Not a problem, but @clalancette noticed the version listed here doesn't exist in this release repo.